spec.md

```diff
@@ -20,32 +20,28 @@ This Artifacts Spec provides a reference for artifact authors and registry imple
 
 ## Scope
 
-Create a first version of the Artifacts specification that is dependent on [OCI manifest][image-manifest] 1.0 to represent individual artifact types including [OCI Image][image-spec], [Helm][helm] and [Singularity][singularity].
-
-**v1.0 of the image-spec** defines index and manifest.
+Create a first version of Artifacts support which is dependent on [OCI manifest][image-manifest] 1.0 to represent individual artifact types including [OCI Image][image-spec], [Helm][helm] and [Singularity][singularity].
 
 <img src="./media/image-spec-objects.png" width=500x>
 
-**Artifacts 1.0** utilize manifest and index to represent multiple artifact types, including the [OCI Image-spec][image-spec]
-
 <img src="./media/artifacts-spec-objects.png" width=500x>
 
 ### Future Scope
 
-Future versions of this specification will support new artifact types representing collections of artifacts using [OCI Index][image-index]. A means to identify an index as a type of artifact will be required.
+Future versions will support new artifact types representing collections of artifacts using [OCI Index][image-index]. A means to identify an index as a type of artifact will be required.
```

Comment:
not sure how much this helps really.. can we just point to some open issues / pull requests in the image and distribution repos? maybe summarize what may be coming in future revisions of this reference?

Reply:
The CNAB community is patiently waiting for us to add index support. We know the work we're proposing, we just want to wrap up this iteration before addressing the versioning of index. 
So, I'd say it helps identify a goal. Does it hurt to keep in?

Reply:
The CNAB community is patiently waiting for Index support. We've agreed to complete the first iteration of artifacts before we come back to versioning index with a config object. 
Keeping this demonstrates our commitment and their users understand the goals, without explicitly calling out the solution. 
So, does it hurt to keep it in, giving value to the scope we have now and the future?

Reply:
looks good we can track a list of open dependencies in in the issues list..